### PR TITLE
Fetch user role from JWT. Remove ADMIN env variable

### PR DESCRIPTION
**Pull request by vpetrusenko**
* Remove ADMIN filed from ENV
* Add role to members table
* Bind role from JWT

### CODE REVIEW

File: spec/api/api_v2/auth/jwt_authenticator_spec.rb [hunk 1]
```diff
@@ -129,7 +134,7 @@
       end
 
       it 'should register new member and return instance' do
```
Comment (ysv) [inline spec/api/api_v2/auth/jwt_authenticator_spec.rb:136]: @vpetrusenko can you add spec where we first create Member with role :member from barong JWT and then we receive new JWT where role is :admin

File: app/models/member.rb [hunk 2]
```diff
@@ -53,10 +53,6 @@ def current=(user)
       Thread.current[:user] = user
     end
```
Comment (ysv) [inline app/models/member.rb:61]: add
```ruby
validates :role, inclusion: { in: -> (_) { Member.roles.map(&:to_s) } }
```
```ruby
class << self
  def roles
      %i[member admin].freeze
  end
end

def role
  role&.inquiry
end
```

**Review by ysv — CHANGES_REQUESTED** [review]

**Comment by ysv** [comment]
@vpetrusenko https://github.com/rubykube/peatio/pull/1694/files#diff-99215219f23b3e704ba1b4ecadb52a71R32

```member.role     = info_hash.fetch('role')```

File: app/models/member.rb [hunk 3]
```diff
@@ -79,12 +77,22 @@ def locate_email(auth_hash)
...
   end
 
+  class << self
+    def roles
+      %i[member admin].freeze
+    end
+  end
+
+  def role
+    super.inquiry
```
Comment (ysv) [inline app/models/member.rb]: super&

File: db/schema.rb [hunk 4]
```diff
@@ -120,13 +120,14 @@
   add_index "markets", ["position"], name: "index_markets_on_position", using: :btree
 
   create_table "members", force: :cascade do |t|
```
Comment (mod) [inline db/schema.rb:122]: should we list UID in here ?

File: app/models/member.rb [hunk 5]
```diff
@@ -45,6 +47,10 @@ def from_auth(auth_hash)
       raise e
     end
 
+    def roles
+      %i[member admin].freeze
```
Comment (mod) [inline app/models/member.rb:51]: I suggest adding accountant and operator roles here

**Review by mod — CHANGES_REQUESTED** [review]

File: db/schema.rb (hunk 4 above)
Comment (ysv) [reply db/schema.rb:122]: @mod but we don't use UID in peatio
we only save it in authentications

File: app/models/member.rb (hunk 5 above)
Comment (vpetrusenko) [reply app/models/member.rb:51]: @mod i would like to add this roles in separate accountant patch

**Pull request closed without merging**